src/pages/api/arbeidsgiver/tilgang/index.ts

```diff
@@ -22,7 +22,7 @@ const handler = async (
     res.status(200).json(getMockDb(req).tilgang);
   } else {
     const tokenX = await getSyfoOppfolgingsplanserviceTokenFromRequest(req);
```

Comment:
Jeg tror du må endre denne til `getOppfolgingsplanBackendTokenFromRequest` for at det skal fungere.

Reply:
Jeg har tatt meg den frihet å endre dette og oppdatere branchen fra main siden jeg trengte branchen for å teste endringer i oppfolgingsplan-backend og syfobrukertilgang :)